product-service/src/handlers/catalogBatchProcess.ts

```diff
@@ -0,0 +1,58 @@
+import { SNS } from 'aws-sdk';
```

Comment:
It's better use `aws-sdk` v3:
https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-sns/index.html